src/poll.ts

```diff
@@ -236,17 +250,53 @@ namespace Poll
 
 			return null;
 		}
+		
+		if (!voicePoll) 
+		{
+			voiceChannel = null;
+		}
 
 		return new Poll
 		(
 			message,
 			desc + ' ' + command[1],
 
 			() => action(target),
-			() => (target.voiceChannelID === message.member.voiceChannelID),
-			() => Math.floor(voiceChannel.members.array().length * fraction)
+			() => (true),
+			() => 
+			{
+				let userCount: number = 0;
+				if (voicePoll)
+				{
+					userCount = voiceChannel.members.array().length;
+				}
+				else
+				{
+					let botCount: number = 0;
+					for (let member of server.members.array())
+					{
+						if (member.user.presence.status === "online" && !member.user.bot === true)
+						{
+							userCount ++;
+						}
+						else if (member.user.bot === true)
+						{
+							botCount ++;
+						}
+					}
+					
+					if (userCount < (server.members.array().length - botCount) * (2/3))
```

Comment:
What is this condition for? Shouldn't `userCount` already be correct?

Reply:
this is how the userCount is capped. If the userCount is too low, it will be set to at least 2/3 of sever size